spec/support/api_helper.rb

```diff
@@ -112,6 +112,12 @@ def stub_api_action_role(collection, action_type, method, action, identifier)
         allow(Api::ApiConfig.collections[collection][action_type]).to receive(method) { updated_method }
       end
 
+      def stub_api_collection_config(collection, attr, value)
+        collection_config = Api::ApiConfig.collections[collection].dup
+        collection_config[attr] = value
+        allow(Api::ApiConfig.collections).to receive(collection) { collection_config }
```

Comment:
This seems like it is stubbing a crucial part of the SUT. Would it be possible to stub closer to the boundary? Something like: 
```ruby
settings_stub = ::Config::Options.new.tap do |o|
  o.add_source!({:foo => "bar"})
  o.load!
end

stub_const(::Settings, settings_stub)
```